recipe/meta.yaml

```diff
@@ -34,18 +36,29 @@ requirements:
     - toolchain
     - boost-cpp 1.63.*
     - cmake
-    - zlib 1.2.*
-    - snappy
-    - brotli
-    - thrift-cpp
+    - zlib 1.2.*        # [unix]
+    - snappy            # [unix]
+    - brotli            # [unix]
+    - thrift-cpp        # [unix]
+    - python            # [win]
```

Comment:
Is there an upstream JIRA for fixing the _HOME variables? Ideally we want to be building with the conda-forge toolchain for consistency